tests/e2e/csi_snapshot_utils.go

```diff
@@ -341,9 +341,12 @@ func getVolumeSnapshotIdFromSnapshotHandle(ctx context.Context,
 	var snapshotID string
 	var snapshotHandle string
 	var err error
+
 	if vanillaCluster || supervisorCluster {
 		snapshotHandle = *snapshotContent.Status.SnapshotHandle
 		snapshotID = strings.Split(snapshotHandle, "+")[1]
+		framework.Logf("********** snapshotHandle :%v", snapshotHandle)
```

Comment:
Please remove unnecessary log and commented lines across all files

Reply:
done